dirty.html

```diff
@@ -1,13 +1,11 @@
 <html>  
   <head>
-    <h1>WDI London September 2013
+    <h1>WDI London September 2013</h1>
```

Comment:
`<title>` better, no h1, h2, etc in the head!

https://developer.mozilla.org/en/docs/Web/HTML/Element/title